### PR TITLE
hotfixes for F8

### PR DESCRIPTION
**Pull request by sonalk0209**
Signed-off-by: Sonal Kalbende <sonal.kalbende@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by rahul-hatwar — APPROVED** [review]
LGTM

File: tests/ha/test_cluster_shutdown_start.py [hunk 1]
```diff
@@ -626,6 +625,9 @@ def test_write_after_cluster_restart(self):
         LOGGER.info("Step 3: Cluster restarted fine and all Pods online.")
         LOGGER.info("STEP 4: Perform WRITEs with variable object sizes. 0B + (1KB - 512MB). "
                     "Verify READs and DI on the written data.")
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.test_prefix = 'test-29470-1'
+        self.s3_clean.update(users)
```
Comment (ApurwaMohite) [inline tests/ha/test_cluster_shutdown_start.py:630]: Should skip read also?

**Review by ApurwaMohite — APPROVED** [review]

**Review by ankita-93 — APPROVED** [review]
lgtm

File: tests/ha/test_cluster_shutdown_start.py (hunk 1 above)
Comment (sonalk0209) [reply tests/ha/test_cluster_shutdown_start.py:630]: No this is actually after cluster restarted, so read can be done.

**Review by ravindrachoudhari — APPROVED** [review]

**Pull request merged**